devel/clucene/Portfile

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem 1.0
-PortGroup               cmake 1.0
+PortGroup               cmake 1.1
 PortGroup               conflicts_build 1.0
 PortGroup               muniversal 1.0
 
 name                    clucene
 version                 2.3.3.4
-revision                2
+revision                0
```

Comment:
Since you're not updating the version number, I would've thought that the revision number should be incremented to 3 rather than set to 0.

Reply:
@harens thanks, this was a typo based on a lot of rebase.